robotbase/src/main/java/com/team2357/frc2023/subsystems/ArmRotationSubsystem.java

```diff
@@ -0,0 +1,45 @@
+package com.team2357.frc2023.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.team2357.frc2023.Constants;
+import com.team2357.frc2023.Robot;
+
+import edu.wpi.first.wpilibj.RobotState;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ArmRotationSubsystem extends SubsystemBase {
+  private CANSparkMax m_rotationMotor;
+
+  public ArmRotationSubsystem() {
+    m_rotationMotor = new CANSparkMax(Constants.CAN_ID.ARM_ROTATION_MOTOR, MotorType.kBrushless);
+
+    m_rotationMotor.setSmartCurrentLimit(60, 30);
```

Comment:
60 and 30 should be constants. How did you determine these numbers, they are pretty high guesses?